src/test/java/Home20work.java

```diff
@@ -1,30 +1,38 @@
 import org.openqa.selenium.By;
+import org.openqa.selenium.ElementClickInterceptedException;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
 import org.testng.annotations.Test;
 
 import java.time.Duration;
 
 
 public class Home20work {
+    static WebDriverWait wait=null;
+
     @Test
-    public void clearCurrentQueue() {
-        starup();
-        //login();
-    }
+    private static void registration1() {
 
-    private void starup() {
         WebDriver driver = new ChromeDriver();
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        //driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10))
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
         String url = "https://bbb.testpro.io/";
         driver.get(url);
-        WebElement emailField = driver.findElement(By.cssSelector("[type='email']"));
-        emailField.clear();
-        emailField.sendKeys("karkisuman0330@gmail.com");
-        WebElement passwordField = driver.findElement(By.cssSelector("[type='password']"));
-        passwordField.clear();
-        passwordField.sendKeys("te$t$tudent");
+
+        WebElement registrationButton1 = wait.until(ExpectedConditions.elementToBeClickable(By.id("[id='hel']")));
+        //driver.findElement(By.id("[id='hel']"));
+        registrationButton1.click();
+
+
+        WebElement registerButton1 = driver.findElement(By.id("[id='button']"));
+        Assert.assertTrue(registerButton1.isDisplayed());
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._